src/routers/mesh_gateway_router.ts

```diff
@@ -0,0 +1,48 @@
+import * as express from 'express';
+import * as asyncHandler from 'express-async-handler';
+// tslint:disable-next-line:no-implicit-dependencies
+import 'reflect-metadata';
+
+import { MeshGatewayHandlers } from '../handlers/mesh_gateway_handlers';
+import { OrderBookService } from '../services/orderbook_service';
+
+export const createMeshGatewayRouter = (orderBook: OrderBookService): express.Router => {
+    const router = express.Router();
+    const handlers = new MeshGatewayHandlers(orderBook);
+    /**
+     * GET AssetPairs endpoint retrieves a list of available asset pairs and the information required to trade them.
+     * http://sra-spec.s3-website-us-east-1.amazonaws.com/#operation/getAssetPairs
+     */
+    router.get('/asset_pairs', asyncHandler(MeshGatewayHandlers.assetPairsAsync.bind(MeshGatewayHandlers)));
```

Comment:
why is a static method being used here but not for the others?

Reply:
They are static if they do not use `this._orderBook` and instance methods otherwise